### PR TITLE
change ip4 type to list of str

### PR DESCRIPTION
**Pull request by haddystuff**
##### SUMMARY
This PR adds support of multiple ipv4 addresses on one interface which was requested here #1088.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
plugins/modules/net_tools/nmcli.py

##### ADDITIONAL INFORMATION
As it turns out changing type of parameter is enough to make it work.
Now list can be used in ```ip4``` parameter in playbook, but string is also working, so it's backwards compatible.
I didn't add any additional test or documentation just for now. I think such change need to be discussed first.

### CODE REVIEW

**Comment by felixfontein** [comment]
Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to)? Thanks.

**Comment by haddystuff** [comment]
ok, let me just add some tests and examples to documentation

**Comment by haddystuff** [comment]
Adding multiple addresses support for ip6 request more time to implement, so i'll make another pull request later on.

File: changelogs/fragments/1088-nmcli_add_multiple_addresses_support.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+minor_changes:
+  - add multiple addresses support for ip4 parameter
```
Comment (Ajpantuso) [inline changelogs/fragments/1088-nmcli_add_multiple_addresses_support.yml:3]:
```suggestion
  - nmcli - add multiple addresses support for ``ip4`` parameter
```

**Review by Ajpantuso — CHANGES_REQUESTED** [review]
Other than the changelog fragment this looks good. Pulled this locally for testing and haven't found any issues.

**Review by Ajpantuso — APPROVED** [review]
LGTM

**Comment by felixfontein** [comment]
@haddystuff thanks for implementing this!
@Ajpantuso thanks for reviewing!

**Pull request merged**